data/utils/defectsFromBias.py

```diff
@@ -94,6 +95,6 @@ def writeDefectsFile(bboxList, path, detectorSerial, detectorName):
 
     test2BBoxList = getBBoxList(DefectsPath, detectorName)
     assert len(bboxList) == len(test2BBoxList)
-    for boxA, boxB in itertools.izip(bboxList, test2BBoxList):
+    for boxA, boxB in zip(bboxList, test2BBoxList):
```

Comment:
might be worth adding `from builtins import zip` at the top to ensure we get the iterator version on python2.